tests/scalers/new-relic.test.ts

```diff
@@ -0,0 +1,257 @@
+  t.is('5', replicaCount, 'Replica count should be maxed at 5')
+
+  for (let i = 0; i < 50 && replicaCount !== '0'; i++) {
+    replicaCount = sh.exec(
+      `kubectl get deployment.apps/keda-test-app --namespace ${testNamespace} -o jsonpath="{.spec.replicas}"`
+    ).stdout
+    if (replicaCount !== '0') {
+      sh.exec('sleep 5s')
+    }
+  }
+
+  t.is('0', replicaCount, 'Replica count should be 0 after 3 minutes')
+  sh.exec('sleep 10s')
+})
+
+
+test.after.always.cb('clean up newrelic resources', t => {
+  sh.exec(`helm delete --namespace ${newRelicNamespace} nri-keda`)
+  sh.exec(`kubectl delete namespace ${newRelicNamespace} --force`)
+  sh.exec(`kubectl delete namespace ${testNamespace} --force`)
+  t.end()
+})
+
+const generateRequestsYaml = `apiVersion: batch/v1
+kind: Job
+metadata:
+  name: generate-requests
+spec:
+  template:
+    spec:
+      containers:
+      - image: jordi/ab
+        name: test
+        command: ["/bin/sh"]
+        args: ["-c", "for i in $(seq 1 60);do echo $i;ab -c 5 -n 1000 -v 2 http://test-app/;sleep 1;done"]
+      restartPolicy: Never
+  activeDeadlineSeconds: 120
+  backoffLimit: 2`
+
+const deployYaml = `apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test-app
+  name: test-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test-app
+  template:
+    metadata:
+      labels:
+        app: test-app
+        type: keda-testing
+    spec:
+      containers:
+      - name: prom-test-app
+        image: tbickford/simple-web-app-prometheus:a13ade9
+        imagePullPolicy: IfNotPresent
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: keda-test-app
+  name: keda-test-app
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      app: keda-test-app
+  template:
+    metadata:
+      labels:
+        app: keda-test-app
+        type: keda-testing
+    spec:
+      containers:
+      - name: prom-test-app
+        image: tbickford/simple-web-app-prometheus:a13ade9
+        imagePullPolicy: IfNotPresent
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: test-app
+  annotations:
+    prometheus.io/scrape: "true"
+  name: test-app
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    type: keda-testing
+---
+apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: newrelic-trigger
+spec:
+  secretTargetRef:
+  - parameter: queryKey
+    name: newrelic-secret
+    key: newRelicApiKey
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: newrelic-secret
+type: Opaque
+data:
+  newRelicApiKey: {{NEWRELIC_API_KEY}}
+---
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: new-relic-scaledobject
+spec:
+  scaleTargetRef:
+    name: keda-test-app
+  minReplicaCount: 0
+  maxReplicaCount: 5
+  pollingInterval: 5
+  cooldownPeriod:  10
+  triggers:
+  - type: new-relic
+    metadata:
+      account: '{{NEWRELIC_ACCOUNT_ID}}'
+      threshold: '100'
```

Comment:
```suggestion
      region: 'EU'
      threshold: '100'
```

Reply:
Fixed, I added as a variable, so local test can be run on either US or EU region, but by default will use EU region